### PR TITLE
30 create input component

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#30
and the info card from #40 
=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/components/Cards/TherapyGoals/style.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import styled from '@emotion/styled';
+import setMargin from '../../../helpers/set-margin';
+
+export const CardWrapper = styled.div`
+  width: 100%;
```
Comment (dupreesi) [inline client/src/components/Cards/TherapyGoals/style.js:5]: do we need setMargin here, too?

**Review by dupreesi — CHANGES_REQUESTED** [review]
great work as usual @RamyAlshurafa  :) 

can you check the following bits pls:
- [ ] on storybook on Therapy Goals Card I can only type once into the textarea before it looses focus (on textarea input it works)

File: client/src/components/Cards/TherapyGoals/index.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+    handleChange([...goals, { goal: '', category: '' }]);
+  };
+
+  const handleRemove = (i) => {
+    handleChange(goals.filter((_, _i) => _i !== i));
+  };
+
+  const handleGoalChange = (value, i) => {
+    const newGaols = [...goals];
```
[the remark this answers is not in the document]
Comment (RamyAlshurafa) [reply client/src/components/Cards/TherapyGoals/index.js:42]: done

File: client/src/components/Cards/TherapyGoals/style.js (hunk 1 above)
Comment (RamyAlshurafa) [reply client/src/components/Cards/TherapyGoals/style.js:5]: i made a wrapper that accepts margins

**Review by dupreesi — APPROVED** [review]

**Pull request merged**